gamestonk_terminal/portfolio/port_menu.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+from matplotlib import pyplot as plt
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.portfolio import rh_api
+
+
+def print_port(show_login):
+
+    """ Print help """
+
+    print("\nPortfolio:")
+    print("   help          show this menu again")
+    print("   q             quit this menu, and shows back to main menu")
+    print("   quit          quit to abandon program")
+    print(" ")
+    print("Robinhood:")
+    print("")
+    if show_login:
+        print("   login         login")
+    print("   hold          look at current holdings")
+    print("   rhhist        look at historical portfolio")
+    print(" ")
+
+
+def port_menu():
```

Comment:
Probably too late for this, but we are slowly changing the "architecture" of the code, to be more readable and modular.

See how https://github.com/DidierRLopes/GamestonkTerminal/blob/main/gamestonk_terminal/behavioural_analysis/ba_controller.py has been refactored.